web/core/local-db/worker/wa-sqlite/src/sqlite-api.js

```
@@ -0,0 +1,899 @@
+    Module.update_hook(db, adapt(xUpdateHook));
+  };
+
+  sqlite3.value = function (pValue) {
+    const type = sqlite3.value_type(pValue);
+    switch (type) {
+      case SQLite.SQLITE_BLOB:
+        return sqlite3.value_blob(pValue);
+      case SQLite.SQLITE_FLOAT:
+        return sqlite3.value_double(pValue);
+      case SQLite.SQLITE_INTEGER:
+        const lo32 = sqlite3.value_int(pValue);
```

Biome

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)